src/Components/App.tsx

```diff
@@ -565,14 +568,25 @@ const FontPreload = styled.span`
 `;
 
 export default function App() {
+  const { t } = useTranslation();
   const evaluateHandlerRef = useRef(noop);
+
+  useEffect(() => {
+    document.documentElement.lang = i18n.language;
+    i18n.on("languageChanged", lng => {
+      document.documentElement.lang = lng;
+    });
+
+    document.title = t("tshetUinhAutoderiver");
```

Comment:
I think this should be put into `i18n.ts` since nothing is related to the `App` component.
```suggestion
    document.documentElement.lang = i18n.language;
    document.title = i18n.t("tshetUinhAutoderiver");
    i18n.on("languageChanged", lng => {
      document.documentElement.lang = lng;
      document.title = i18n.t("tshetUinhAutoderiver");
    });
```